src/Simulation/Native/src/simulator/wavefunction.hpp

```diff
@@ -90,6 +91,52 @@ class Wavefunction
 
     void flush() const
     {
+        //logic to reorder
```

Comment:
style: the common convention is to have a single space between `//` and the text that follows in text comments, and no space when commenting out code